DAGs overview page + details (#2121)
astronomer/docs: astro/manage-dags.md

| User action                              | **DAGs** page workflow                                                 | Equivalent Airflow UI workflow                                                                                                  |
| ---------------------------------------- | ---------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------- |
| Trigger a DAG run                        | Click **Run**                                                          | Click the **Play** icon on the **DAGs** page.                                                                                   |
| View the DAG run grid                    | None. DAG code appears by default.                                     | Click the DAG name on the **DAGs** page.                                                                                        |

> | Retry a DAG/ task run.                   | Click the DAG/task run in the DAG run grid, then click **Retry**.      | Click the DAG name on the **DAGs** page, click the DAG/task run in the **Grid** view, then click **Clear existing tasks**.      |

Reviewer: An unknown: Is this right? There are two different ways to retry this:
![Screenshot 2023-05-15 at 11 59 47 AM](https://github.com/astronomer/docs/assets/74574233/99fd31e9-f3d9-443a-a5b8-6d5f96291648)

Reviewer 2: this is for Retrying the DAG run in the Airflow UI, seems like in Cloud UI, we can just "Retry a DAG - which is equivalent to Clear existing tasks". 

And for clearing the task runs, options are similar, except Recursive seems to be missing and by default it clears the current task and it's child tasks only.
<img width="448" alt="Screenshot 2023-05-15 at 5 50 05 PM" src="https://github.com/astronomer/docs/assets/75730393/7dd67260-2b79-4dbb-9401-f35a7f5d6102">

<img width="865" alt="Screenshot 2023-05-15 at 5 50 17 PM" src="https://github.com/astronomer/docs/assets/75730393/baff5479-863f-4a3c-afb7-683bd9a79a2a">

Reviewer: Oh! I was afraid to click Retry in dev in case it automatically retried a DAG I didn't own 😅

> | Retry a DAG/ task run.                   | Click the DAG/task run in the DAG run grid, then click **Retry**.      | Click the DAG name on the **DAGs** page, click the DAG/task run in the **Grid** view, then click **Clear existing tasks**.      |

Reviewer: ```suggestion
| Retry a DAG/ task run.                   | Click the DAG/task run in the DAG run grid, then click **Retry**.      | Click the DAG name on the **DAGs** page, click the DAG/task run in the **Grid** view, then click **Clear existing tasks**/ **Clear**.      |
```

Reviewer 2: @jwitz it should be click "Retry" because there is no "Clear" button in the Cloud UI

> | Retry a DAG/ task run.                   | Click the DAG/task run in the DAG run grid, then click **Retry**.      | Click the DAG name on the **DAGs** page, click the DAG/task run in the **Grid** view, then click **Clear existing tasks**.      |

Reviewer: ```suggestion
| Re-run a DAG/ task run.                   | Click the DAG/task run in the DAG run grid, then click **Retry**.      | Click the DAG name on the **DAGs** page, click the DAG/task run in the **Grid** view, then click **Clear existing tasks**.      |
```

> | Mark a DAG/ task run as success/ failed. | Click the DAG/task run in the DAG run grid, then click **Mark as...**. | Click the DAG name on the **DAGs** page, click the DAG/task run in the **Grid** view, then click **Mark Success/ Mark Failed**. |

Reviewer: An unknown: what filters do we apply to this action? The Airflow UI has many options which aren't visible here.
![Screenshot 2023-05-15 at 10 52 56 AM](https://github.com/astronomer/docs/assets/74574233/befa9da3-62e7-4a61-92ba-7df087db02a3)

Reviewer 2: @jwitz they are visible but on the "Task" Retry

> ---

Reviewer 3: This page now has the Airflow Graph as well, sorry if I missed it but seems that may need to be mentioned here too?

sidebar_label: 'Manage DAG runs from the Cloud UI'
title: "Manage DAGs"
id: manage-dags
---

> 

Reviewer: ```suggestion

:::info

Managing DAGs from the Cloud UI is currently available only on [Astro Hosted](astro-overview.md). Support on Astro Hybrid is coming soon.

:::
```

---

> sidebar_label: 'Manage DAG runs from the Cloud UI'
> title: "Manage DAGs"

Reviewer: ```suggestion
sidebar_label: 'Manage DAGs'
title: "Manage DAG runs from the Cloud UI"
```
